packages/gatsby/src/redux/actions/public.js

```diff
@@ -1424,16 +1424,20 @@ actions.createServerVisitedPage = (chunkName: string) => {
  * @param {Object} manifest Manifest data
  * @param {string} manifest.manifestId An id which ties the revision unique state of this manifest to the unique revision state of a data source.
  * @param {Object} manifest.node The Gatsyby node to tie the manifestId to. See the "createNode" action for more information about the node object details.
+ * @param {string} manifest.updatedAtUTC (optional) The time in which the node was last updated
+ * @param {number} manifest.maxDaysOld (optional) Choose the days since a node was last updated (ie create manifests for nodes that were last updated up to 30 days)
```

Comment:
```suggestion
 * @param {number} manifest.maxDaysOld (optional) Choose the days since a node was last updated (e.g. create manifests for nodes that were last updated up to 30 days)
```